### ISSUE TITLE
Home Assistant doesn't support negative values?

### ISSUE THREAD

**Issue by OmgImAlexis** [madpilot/hass-amber-electric#24]
Might need to talk to their devs on fixing this as negative prices should be supported.

```
The following entities have a negative state while a positive state is expected:
sensor.home_general_energy_price (-0.06)
```

**Comment by madpilot** [comment]
Ah, good catch. I'll ping them on Discord.

**Comment by madpilot** [comment]
Oh, good - you've added a issue in their repo 🎉

**Comment by madpilot** [comment]
Closing - issue is in home assistant

**Comment by OmgImAlexis** [comment]
Fixed in Home Assistant Core `2021.9.5`.
